include/parser/independent/expression-compiler-clike.hpp

```diff
@@ -28,6 +28,15 @@
 #include "parser/independent/integer-parser.hpp"
 #include "parser/independent/string-parser.hpp"
 
+#define clike_recordError(errorList, position, msg, ...) \
```

Comment:
Due to the translation concept of Qt consisting mostly of macros (e.g. `QT_TRANSLATE_NOOP`) we use macros to simplify the expression around the message to be marked for translation. So in my opinion this is fine.
But it should be ALL_CAPS.